### DISCUSSION TITLE
Odd blue corner on widgets (transparent window)

### DISCUSSION THREAD

**Discussion by ItsVoltz** [AllenDang/giu#139] [Q&A] [answered]
So I have been playing around with giu and my window is slightly transparent (single window) and for some reason the corners and that on my widgets are slightly transparent enough to where you can see thru and it makes the corners have these ugly corners depending on the color of the window / desktop behind it.

Note: I am using Nicify's appleWindowProperties func and his WindowMove hook, I removed the part that sets window opacity in the window properties and the window is still slightly transparent

![test-window](https://user-images.githubusercontent.com/29076540/107001830-51144d00-6758-11eb-9205-0ba3601d52a5.png)

**Comment by Rel1cx** [comment, marked as answer]
The problem is that the default value of framebordersize is not appropriate when frameborder is rounded, if you want to get pixel perfect display with rounded corners it is recommended to set `StyleVarFrameBorderSize` to 0
in loop func:
```golang
defer imgui.PopStyleVar()
imgui.PushStyleVarFloat(imgui.StyleVarFrameBorderSize, 0)
```

**Reply by ItsVoltz** [reply]
Worked thank you so much you're awesome!
